### PR TITLE
Remove IdSet class

### PR DESCRIPTION
**Pull request by rshunim**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10014

### CODE REVIEW

File: Tests/scripts/collect_tests/collect_tests.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def __init__(
             version_range: VersionRange | None,
             reason_description: str,
             conf: TestConf | None,
-            id_set: IdSet | Graph | None,
+            id_set: Graph | None,
```
Comment (dorschw) [inline Tests/scripts/collect_tests/collect_tests.py:89]: let's change `id_set` to `graph` everywhere

File: Tests/scripts/collect_tests/collect_tests.py [hunk 2]
```diff
@@ -302,13 +302,10 @@ def __bool__(self):
...
-    def __init__(self, marketplace: MarketplaceVersions, graph: bool = False):
+    def __init__(self, marketplace: MarketplaceVersions):
         self.marketplace = marketplace
-        self.id_set: IdSet | Graph
-        if graph:
-            self.id_set = Graph(marketplace)
-        else:
-            self.id_set = IdSet(marketplace, PATHS.id_set_path)
+        self.id_set: Graph
+        self.id_set = Graph(marketplace)
```
Comment (dorschw) [inline Tests/scripts/collect_tests/collect_tests.py:308]:
```suggestion
        self.id_set = Graph(marketplace)
```

**Review by dorschw — COMMENTED** [review]
Very nice
1. See the very small comments
2. Run a nightly on this branch and link to it in a comment here, before merging

**Comment by rshunim** [comment]
link to [nightly](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/900725)

**Pull request closed without merging**